src/porepy/fracs/fractures_2d.py

```diff
@@ -232,16 +235,26 @@ def mesh(
             # Delete constraints corresponding to deleted edges
             constraints = np.delete(constraints, to_delete)
 
+        # Consider the dimension of the problem, normally 2d but if dfn it's a 1d
```

Comment:
Please no 'it's'  in the documentation. I think what they tried to teach me in school has finally sunk in.